src/components/FormikAutocomplete.tsx

```diff
@@ -47,21 +49,18 @@ export default function FormikAutocomplete(props: FAProps): JSX.Element {
         id={id}
         multiple
         filterOptions={filterOptions}
-        options={values}
         value={values}
         inputValue={inputValue}
-        onChange={(e, val) =>
-          onChange ? onChange(val) : formik.setFieldValue(id, val)
-        }
-        onInputChange={(event, newInputValue) => {
+        onChange={onChange}
+        onInputChange={(e, newInputValue) => {
           const options = newInputValue.split(',');
 
           if (options.length > 1) {
             const vals = values.concat(
               options.map((x) => x.trim()).filter((x) => x)
             );
             setInputValue('');
-            return onChange ? onChange(vals) : formik.setFieldValue(id, vals);
+            return onChange(e, vals, 'selectOption');
```

Comment:
what does the 'selectOptions' do?

Reply:
The third parameter is required. Out of "createOption", "selectOption", "removeOption", "blur" or "clear" it seemed the most logical in this case.